pkg/tbtc/chain.go

```diff
@@ -359,6 +298,7 @@ type WalletCoordinatorChain interface {
 	// proposal events according to the provided filter or unfiltered if the
 	// filter is nil. Returned events are sorted by the block number in the
 	// ascending order, i.e. the latest event is at the end of the slice.
+	// TODO: This function is not used, consider removing it.
 	PastDepositSweepProposalSubmittedEvents(
```

Comment:
@lukasz-zimnoch Can we safely remove it? This function is not used, but there is some existing local chain implementation, which I'm not sure I can remove.

Reply:
Yeah I think we can remove it.

Reply:
da06660aefb76b9a0da07a385e1607bd74ff1dbe